src/views/List.jsx

```diff
@@ -16,7 +17,6 @@ export function List({ data, listToken }) {
 	);
 
 	const sortedList = comparePurchaseUrgency(data);
```

Comment:
Here you should sort the `filteredListItems` not the `data`, that's why search isn't working any more :slightly_smiling_face: . So it should be